timed/employment/filters.py

```diff
@@ -53,13 +54,20 @@ class UserFilterSet(FilterSet):
     active      = NumberFilter(field_name='is_active')
     supervisor  = NumberFilter(field_name='supervisors')
     is_reviewer = NumberFilter(method='filter_reviewers')
+    is_supervisor = NumberFilter(method='filter_supervisors')
 
     def filter_reviewers(self, queryset, name, value):
         reviewer = Project.objects.filter(reviewers=OuterRef('pk'))
         return queryset.annotate(
             is_reviewer=Exists(reviewer)
         ).filter(is_reviewer=value)
 
+    def filter_supervisors(self, queryset, name, value):
```

Comment:
Didn't see this before but this comment also concerns filter_reviewers:
I think the function name should be called like the filter is called. Otherwise is there a confusion with `supervisor`.

In this example it would be `filter_is_supervisor`